README.md

```diff
@@ -63,22 +63,20 @@ const actionCreator = () => ({
 });
 ```
 
-It is also possible to use a function:
+If you include [thunk middleware](https://github.com/gaearon/redux-thunk) in your middleware stack, it is also possible to use a function to dispatch multiple actions:
 
 ```js
 const actionCreator = () => ({
   type: 'FIRST_ACTION_TYPE',
   payload: {
-    promise: Promise.resolve((action, dispatch, getState) => {
-      dispatch({ type: 'SECEOND_ACTION_TYPE', payload: ... })
+    promise: Promise.resolve((dispatch, getState) => {
```

Comment:
We use a modified version of thunk that is pre-bound to the second action for convenience. Why did you remove this? Please look at the tests :)